lib/vintage_net_lte/modems/quectel_BG96.ex

```diff
@@ -68,7 +68,7 @@ defmodule VintageNetLTE.Modems.QuectelBG96 do
     OK ATQ0
 
     # Define PDP context
-    OK AT+CGDCONT=1,"IP","#{provider_info.apn}"
+    OK AT+CGDCONT=1,"IP","wireless.twilio.com"
```

Comment:
Are you thinking of having a service provider to APN lookup in a future commit? The reason I'm asking is that this is slightly unfortunate here to go back to hardcoding APNs.

Reply:
Thought since we are specifying the service provider explicitly, each modem will know which chatscript to generate from that particular provider for that particular modem?

If we start passing around provider information we will need to figure out the best way to handle wanting to configure many APNs, as this case has came already. The current known case is 2 apns which one does some modem specific configuration for LTE connections.

Reply:
In trying to get my local project running I stumbled upon evidence suggesting vodafone (EU/Germany/I don't know) at least used to have different APNs for their prepaid line and their contract line. I'm currently using the prepaid SIM with the contract line APN though, so not sure how relevant this is nowadays.

Reply:
I'm wondering if it makes sense to have modules for providers separate from modems. They could hold default config / docs for the provider and functionality to override default config. The modems' modules could then use the provider's module name to device on the callscripts instead of a plain string.

Reply:
I am going to merge this PR now since most of this discussion is going to be flushed out in the next PR.  The main focus of this PR is to add the modem lookup table, next PR will be dealing with providers and handling that well.

I think your suggestion is really close to some of the ideas we had for the next PR.